agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -225,16 +212,39 @@ def set(self, **kwargs):
         for key, value in kwargs.items():
             setattr(self, key, value)
 
+    def dump(self):
+        """Returns all the information about the current version in the configuration.
+
+        Raises:
+            NotImplementedError: _description_
+        """
+
+        raise NotImplementedError()
+
 
-def init(app_name=None, base_name=None, **kwargs):
+def init(
+    app_id: Optional[str] = None,
+    host: Optional[str] = None,
+    api_key: Optional[str] = None,
+    max_workers: Optional[int] = None,
+):
     """Main function to be called by the user to initialize the sdk.
 
     Args:
-        app_name: _description_. Defaults to None.
-        base_name: _description_. Defaults to None.
+        app_id (str): The Id of the app.
+        host (str): The host of the backend server.
+        api_key (str): The API key to use for the backend server.
     """
 
     singleton = AgentaSingleton()
 
-    singleton.init(app_name=app_name, base_name=base_name, **kwargs)
-    set_global(setup=singleton.setup, config=singleton.config)
+    singleton.init(app_id=app_id, host=host, api_key=api_key)
+    tracing = Tracing(
+        host=singleton.host,  # type: ignore
+        app_id=singleton.app_id,  # type: ignore
+        variant_id=singleton.variant_id,  # type: ignore
+        variant_name=singleton.variant_name,
+        api_key=singleton.api_key,
+        max_workers=max_workers,
+    )
+    set_global(setup=singleton.setup, config=singleton.config, tracing=tracing)
```

Comment:
why do we need the set_global for tracing if we already did it in the __init__.py?